Description: Spelling fixes
Author: Christian Kastner <ckk@debian.org>
Author: Tobias Frost <tobi@debian.org>
Bug: https://sourceforge.net/p/avarice/patches/31/
Last-Update: 2016-10-17
Gentoo-redif: Kalin KOZHUHAROV <kalin@thinrope.net>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff -Nru avarice-2.13-/doc/avarice.1 avarice-2.13/doc/avarice.1
--- avarice-2.13-/doc/avarice.1	2017-10-03 00:19:56.101860147 +0200
+++ avarice-2.13/doc/avarice.1	2017-10-03 00:26:37.280153338 +0200
@@ -317,7 +317,7 @@
 Print a list of known devices.
 .TP
 .BR \-L ,\  \-\-write-lockbits \ <ll>
-Write lock bits. The lock byte data must be given in two digit hexidecimal
+Write lock bits. The lock byte data must be given in two digit hexadecimal
 format with zero padding if needed.
 .TP
 .BR \-l ,\  \-\-read-lockbits
@@ -362,7 +362,7 @@
 .BR \-W ,\  \-\-write-fuses \ <eehhll>
 Write fuses bytes. \fBee\fR is the extended fuse byte, \fBhh\fR is the high
 fuse byte and \fBll\fR is the low fuse byte. The fuse byte data must be given
-in two digit hexidecimal format with zero padding if needed. All three bytes
+in two digit hexadecimal format with zero padding if needed. All three bytes
 must currently be given.
 .TP
 .BR \-x ,\  \-\-xmega
diff -Nru avarice-2.13-/src/jtag2misc.cc avarice-2.13/src/jtag2misc.cc
--- avarice-2.13-/src/jtag2misc.cc	2012-11-05 15:16:04.000000000 +0100
+++ avarice-2.13/src/jtag2misc.cc	2017-10-03 00:24:16.119809488 +0200
@@ -56,7 +56,7 @@
     }
     catch (jtag_exception& e)
     {
-        fprintf(stderr, "set paramater command failed: %s\n",
+        fprintf(stderr, "set parameter command failed: %s\n",
                 e.what());
         throw;
     }
@@ -86,12 +86,12 @@
     }
     catch (jtag_exception& e)
     {
-        fprintf(stderr, "get paramater command failed: %s\n",
+        fprintf(stderr, "get parameter command failed: %s\n",
                 e.what());
         throw;
     }
     if (resp[0] != RSP_PARAMETER || respSize <= 1)
-        throw jtag_exception("unexpected response to get paramater command");
+        throw jtag_exception("unexpected response to get parameter command");
 }
 
 
diff -Nru avarice-2.13-/src/jtaggeneric.cc avarice-2.13/src/jtaggeneric.cc
--- avarice-2.13-/src/jtaggeneric.cc	2012-11-05 15:16:04.000000000 +0100
+++ avarice-2.13/src/jtaggeneric.cc	2017-10-03 00:24:16.120809477 +0200
@@ -432,7 +432,7 @@
     c = sscanf(fuses, "%02x%02x%02x", temp+2, temp+1, temp );
     if (c != 3)
     {
-        fprintf(stderr, "Error: Fuses specified are not in hexidecimal");
+        fprintf(stderr, "Error: Fuses specified are not in hexadecimal");
         return;
     }
 
@@ -606,7 +606,7 @@
     c = sscanf(lock, "%02x", temp);
     if (c != 1)
     {
-        fprintf(stderr, "Error: Fuses specified are not in hexidecimal");
+        fprintf(stderr, "Error: Fuses specified are not in hexadecimal");
         return;
     }
 
